Hyper/Controllers/Bots/Spawn/SphericalBotSpawnStrategy.cs

```diff
@@ -7,7 +7,7 @@ namespace Hyper.Controllers.Bots.Spawn;
 
 internal class SphericalBotSpawnStrategy : AbstractBotSpawnStrategy
 {
-    private const int MaxBots = 20;
+    private const int MaxBots = 0;
```

Comment:
I think this is a mistake

Reply:
Bots in are still glitching like mad in spherical, so we can either get rid of them completely or keep trying to fix them

Reply:
I think we should keep them anyways. Also I tried to add health to bots and set health to 0 when bots are hit with a projectile but it wasn't as simple as expected. There is some bug where the bots in Scene.cs do not correspond to all the bots in the game

Reply:
Maybe it's connected